### ISSUE TITLE
IT DOESN'T WORKS ANYMORE UNTIL JUNE 24

### ISSUE THREAD

**Issue by Franklyn123** [manojVivek/medium-unlimited#160]
background error

**Comment by NaimulIslam9m** [comment]
😖

**Comment by ace3** [comment]
@Franklyn123 , you should include what website you want to unlock. 
for all of my medium sites, I have no issues.

**Comment by pavinduLakshan** [comment]
Yes, it works for me too. However, the clap button and the footer is missing in the unlocked articles. Previously I could clap to the unlocked articles, but now it's not possible.

**Comment by pprp** [comment]
Do not work now 😢

**Comment by NikulGoyani369** [comment]
![image](https://user-images.githubusercontent.com/19364203/125297593-a6681200-e327-11eb-8d48-92f88e6a4183.png)

I got this error, has anyone solution for it.
